src/screens/Habitoptions.js

```diff
@@ -1,19 +1,57 @@
 import React from 'react';
-import { View, Text, StyleSheet } from 'react-native';
+import { View, Text, StyleSheet, TouchableOpacity, FlatList } from 'react-native';
+
+const options = [
+  { id: 1, name: "Exercise" },
+  { id: 2, name: "Read a Book" },
+  { id: 3, name: "Drink Water" },
+];
+
+const Habitoptions = ({ navigation }) => {
+  const handleSelectOption = (option) => {
+    //console.log(`Selected option: ${optionName}`); 
```

Comment:
you should not leave comments in the code. if you think they should stay, you need to add extra comment to explain what it does